### ISSUE TITLE
Story manager

### ISSUE THREAD

**Issue by sudojunior** [TinkerStorm/story-engine#3]
Heavily depends on #2 for the database, as well as access control for recognizing a user as an author.

- `create ...` - Creates a new story
- `edit ...` - Edit a story
- `delete ...` - Delete one of your stories
- `list ...` - A list of your stories in the database
- `info ...` - Editor info about a story.

> This sets up the bare bones of how stories would be managed, parts below make this a staged release... which is disappointing, when the feature isn't fully usable for what it can already do.

- `add-step ...` - Add a step to a story
- `remove-step ...` - Remove a step from a story

> To actually build message data is a difficult task, let alone constructing a JSON object by hand...

- `add-route ...` - Add a routing option to a step
- `remove-route ...` - Remove a routing option from a step
- `set-routing ...` - Set the routing options for a step

> Somewhere in there, the probability features would fit in... somewhere, array options aren't supported from what I'm aware of... so that would be a very slow process.
